tests/php/FilenameParsing/MigrationLegacyFileIDHelperTest.php

```diff
@@ -1,11 +1,19 @@
 <?php
 namespace SilverStripe\Assets\Tests\FilenameParsing;
 
-use SilverStripe\Assets\FilenameParsing\LegacyFileIDHelper;
+use SilverStripe\Dev\Deprecation;
 use SilverStripe\Assets\FilenameParsing\ParsedFileID;
+use SilverStripe\Assets\FilenameParsing\LegacyFileIDHelper;
 
 class MigrationLegacyFileIDHelperTest extends FileIDHelperTester
 {
+    protected function setUp(): void
+    {
+        parent::setUp();
+        if (Deprecation::get_is_enabled()) {
+            $this->markTestSkipped('Test calls deprecated code');
```

Comment:
Marking test as skipped in setUp() rather than on individual test*() because this is a strange unit test where the bulk of the tests are in parent class FileIDHelperTester()